src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
 body {
   margin: 0;
 }
+
+.container {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 1vmin dotted #000;
+  position: relative;
+
+  &::before,
+  &::after {
+    content: '';
+    display: block;
+    position: absolute;
+    z-index: -1;
+    bottom: 50%;
+    transform-origin: bottom;
+    animation-name: rotate;
+    animation-iteration-count: infinite;
+  }
+
+  &::before {
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    animation-duration: 60 * 60s;
```

Comment:
The animation-duration property is set to '60 * 60s'. This is not a valid SCSS or CSS syntax for multiplication. You should calculate the duration in seconds and set it directly, e.g., '3600s' for 60 minutes.